### DISCUSSION TITLE
Any News About ECDSA or RSA Libraries?

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#16085] [General]
Just curious on if there was anybody who was working on it or knows something.  Related to, https://github.com/vlang/v/discussions/14644 (Does V has a ECDSA lib). By the way, happen to see this on Rosetta Code (https://rosettacode.org/wiki/RSA_code#V_(Vlang).

**Comment by bstnbuck** [comment]
TLS and RSA are at the top of my to-do list. At the moment I don't have enough free time... This should be easy with mbedTLS ([mbedTLS](https://github.com/vlang/v/tree/master/thirdparty/mbedtls/) though.
For ECDSA I don't know any news, but it would be possible with mbedTLS as well.
#11806

**Reply by Wajinn** [reply]
Thanks for giving more information about this.

**Comment by JalonSolov** [comment]
V has a ecdsa module now.
